### DISCUSSION TITLE
Slider speed and Hitsound Volume sliders in editor.

### DISCUSSION THREAD

**Discussion by remclean** [ppy/osu#19288] [Ideas] [answered]
On thing I found really annoying as a new mapper was slider speed was very painstaking to change. You had to make a new time point for every slider with a different speed. I also thought hitsounds would be a good thing to add even tho I never did much with them.

I made a proof of concept photo, ignore the older editor (maybe move the new addition under the snapping slider?), aswell as the poor editing and miss-spelled volume :)
![proof_of_consept](https://user-images.githubusercontent.com/109771463/180335478-c32740a3-11e3-4acc-bd12-65c406e9c407.png)

**Comment by frenzibyte** [comment, marked as answer]
You should be able to change the speed of each slider via the bubble above the slider (by clicking on it and adjust the slider). Same applies for volume, which have pink bubbles below the slider.

The slider should also copy the velocity from its previous one, but currently it doesn't when separated by a hitcircle, which is an oversight and will be fixed next release.

**Reply by remclean** [reply]
Ok, sorry for not noticing, i'm trying to move into lazer and in my experimentation I some how missed that. Thanks!
